src/foam/nanos/ticket/Ticket.js

```diff
@@ -21,8 +21,16 @@ foam.CLASS({
     'foam.nanos.auth.ServiceProviderAware'
   ],
 
+  topics: [
+    'finished',
+    'throwError'
+  ],
+
   requires: [
     'foam.nanos.ticket.TicketStatus',
+    'foam.dao.AbstractDAO',
```

Comment:
Can you sort these and also imports below.

Reply:
Done.